client/package.json

```diff
@@ -161,6 +161,11 @@
           "markdownDescription": "SDK image to use for cross-compilation and debugging.\n\nExample: `core-image-minimal`",
           "examples": "core-image-minimal"
         },
+        "bitbake.disableTemporaryFiles": {
```

Comment:
I'd rename this, we have other temporary files used for other things (like the mementos to display or not error notifications). => "disableEmbeddedLanguages"

Reply:
That was my first idea, then I realized many features with embedded languages were still working (at least partially), such as completion and hover.

What about `disableEmbeddedLanguagesFiles` ?

Reply:
yes, nice!

Reply:
done